### PR TITLE
po: Add Georgian translation

### PR DESCRIPTION
**Pull request by NorwayFun**
## Pull request checklist

- [ x] Lint and unit tests pass locally with my changes
- [ ] Extended the README / documentation, if necessary

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x ] Other (please describe):
Adding translation
## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

(Write your answer here.)

## Other information

Adding Georgian translation.

### CODE REVIEW

File: po/ka.po [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-31 13:18+0200\n"
+"PO-Revision-Date: 2024-06-27 05:53+0200\n"
+"Last-Translator: Temuri Doghonadze <temuri.doghonadze@gmail.com>\n"
+"Language-Team: Georgian <(nothing)>\n"
+"Language: ka\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```
Comment (kamilprusko) [inline po/ka.po:18]: Please change it to `"Plural-Forms: nplurals=1; plural=0;\n"`

Then, `msgstr[1]` entries can be removed.

File: po/ka.po [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+#: data/resources/menus.ui:6 plugins/actions/enums.vala:188
+#: plugins/actions/resources/action-page.ui:155 src/application.vala:90
+msgid "Start"
+msgstr "გაშვება"
+
+#: data/resources/menus.ui:11 data/resources/window.ui:180
+#: plugins/actions/enums.vala:197 plugins/actions/resources/action-page.ui:200
+#: src/application.vala:99
+msgid "Pause"
+msgstr "გაჩერება"
```
Comment (kamilprusko) [inline po/ka.po:77]: Pause and Stop have same translation. How about using `პაუზა` here (suggestion from Google Translator)?

**Review by kamilprusko — CHANGES_REQUESTED** [review]
Hi @NorwayFun 

Thanks for contributing! Looks good with exception to plural forms and translation for "Pause". Once corrected, I'm happy to merge :)

File: po/ka.po (hunk 1 above)
Comment (NorwayFun) [reply po/ka.po]: It's default for poedit, couldn't convince author to replace it with correct one.
He complained that he's not going to learn "my language" :D

Anyways, corrected now.

**Comment by kamilprusko** [comment]
Thanks! :)

**Pull request merged**